docs/application/toc_all.md

```diff
@@ -258,6 +258,7 @@
 ###### [Grid Layout](/application/dotnet/guides/user-interface/nui/grid-layout.md)
 ###### [Flex Layout](/application/dotnet/guides/user-interface/nui/flex-layout.md)
 ###### [Relative Layout](/application/dotnet/guides/user-interface/nui/relative-layout.md)
+###### [Absolute Layout](/application/dotnet/guides/user-interface/nui/absolute-layout.md)
 ###### [Create Custom Layout](/application/dotnet/guides/user-interface/nui/custom-layout.md)
```

Comment:
Minor issue.

Don't you need to match the order of layouts to the order in https://docs1.stg.tizen.org/staging/1521/application/dotnet/guides/user-interface/nui/layouts/?

In that page the layouts are listed up in Absolute, Linear, Grid, Flex and relative order.
(Custom layout is separted.)

Reply:
Fixed